guides/source/active_record_basics.md

```diff
@@ -220,134 +263,192 @@ The application will have to use the [`id_value`][] alias attribute to access th
+book.author = "J.R.R. Tolkien"
+
+# The above `book` is not yet saved to the database
+
+book.save
 
-A call to `user.save` will commit the record to the database.
+# Now the `book` record is committed to the database.
+```
 
 Finally, if a block is provided, both `create` and `new` will yield the new
 object to that block for initialization, while only `create` will persist
 the resulting object to the database:
 
 ```ruby
-user = User.new do |u|
-  u.name = "David"
-  u.occupation = "Code Artist"
+book = Book.new do |b|
+  b.title = "Metaprogramming Ruby 2"
+  b.author = "Paolo Perrotta"
 end
+
+book.save
+```
+
+The resulting SQL statement from both `book.save` and `Book.create` look something like this:
+
+```sql
+INSERT INTO "books" ("title", "author", "created_at", "updated_at") VALUES (?, ?, ?, ?) RETURNING "id"  [["title", "Metaprogramming Ruby 2"], ["author", "Paolo Perrotta"], ["created_at", "2024-02-22 20:01:18.469952"], ["updated_at", "2024-02-22 20:01:18.469952"]]
 ```
 
 ### Read
 
-Active Record provides a rich API for accessing data within a database. Below
-are a few examples of different data access methods provided by Active Record.
+Active Record provides a rich API for accessing data within a database. You can query a single record or multiple records, filter them by any attribute, order them, group them, select specific fields, and do anything you can do with SQL.
 
 ```ruby
-# return a collection with all users
-users = User.all
+# Return a collection with all books
+books = Book.all
+
+# Return a single book
+first_book = Book.first
+last_book = Book.last
+random_book = Book.take
 ```
 
-```ruby
-# return the first user
-user = User.first
+The above results in the following SQL:
+
+```sql
+# Book.all
+SELECT "books".* FROM "books"
+
+# Book.first
+SELECT "books".* FROM "books" ORDER BY "books"."id" ASC LIMIT ?  [["LIMIT", 1]]
+
+# Book.last
+SELECT "books".* FROM "books" ORDER BY "books"."id" DESC LIMIT ?  [["LIMIT", 1]]
+
+# Book.take
+SELECT "books".* FROM "books" LIMIT ?  [["LIMIT", 1]]
 ```
 
+We can also find specific books with `find_by` and `where`:
+
 ```ruby
-# return the first user named David
-david = User.find_by(name: 'David')
+# Return the first book by the given author
+book = Book.find_by(author: "J.R.R. Tolkien")
+
+# Shorthand for Book.find_by(id: 42)
+book = Book.find(42)
+```
+
+The above resulting in this SQL:
+
+```sql
+SELECT "books".* FROM "books" WHERE "books"."author" = ? LIMIT ?  [["author", "J.R.R. Tolkien"], ["LIMIT", 1]]
+
+SELECT "books".* FROM "books" WHERE "books"."id" = ? LIMIT ?  [["id", 42], ["LIMIT", 1]]
 ```
 
 ```ruby
-# find all users named David who are Code Artists and sort by created_at in reverse chronological order
-users = User.where(name: 'David', occupation: 'Code Artist').order(created_at: :desc)
+# Find all books with a given an author, sort by created_at in reverse chronological order
+hitchhiker_series = Book.where(author: "Douglas Adams").order(created_at: :desc)
```

Comment:
Nit: maybe omitting the variable would shorten the example a bit? I'm not sure how much it adds to the example, and it is not used elsewhere in the next examples.

```suggestion
Book.where(author: "Douglas Adams").order(created_at: :desc)
```